runtime/nvqir/Gates.h

```diff
@@ -114,6 +114,7 @@ getGateByName(GateName name, const std::vector<Scalar> angles = {}) {
     auto theta = angles[0];
     auto phi = angles[1];
     auto lambda = angles[2];
+    // TODO: Confirm matrix representation
```

Comment:
@boschmitt

Reply:
It's wrong, this matrix must be in row-major representation.